ee/modules/160-multitenancy-manager/crds/doc-ru-projects.yaml

```
@@ -10,15 +10,16 @@ spec:
               properties:
                 description:
                   description: Произвольное описание назначения окружения.
-                projectTypeName:
+                templateName:
                   description: |
                     Имя custom resource [ProjectType](cr.html#projecttype), который будет использоваться
                     как шаблон для создания ресурсов окружения.
-                template:
+                parameters:
```

```suggestion
                template:
```
The field cannot be renamed.